lib/index.js

```diff
@@ -200,84 +180,64 @@ const addToNameCal = (food) => {
     </div>`
 }
 
-const prependToBreakfast = (food) => {
-  $('.breakfasts').prepend(`
-    <article class='breakfast' data-breakfast-food-id=${food.id}>` + addToNameCal(food) +
-      `<p class='delete'><button data-breakfast-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
-    </article>`
-  )
-}
-
-const prependToLunch = (food) => {
-  $('.lunches').prepend(`
-    <article class='lunch' data-lunch-food-id=${food.id}>` + addToNameCal(food) +
-      `<p class='delete'><button data-lunch-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
-    </article>`
-  )
-}
-
-const prependToDinner = (food) => {
-    $('.dinners').prepend(`
-      <article class='dinner' data-dinner-food-id=${food.id}>` + addToNameCal(food) +
-        `<p class='delete'><button data-dinner-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
-      </article>`
-    )
-}
-
-const prependToSnack = (food) => {
-  $('.snacks').prepend(`
-    <article class='snack' data-snack-food-id=${food.id}>` + addToNameCal(food) +
-      `<p class='delete'><button data-snack-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
+const prependToMeal = (food, meal) => {
+  let pluralMeal;
+  var mealType = meal.name.toLowerCase()
+  if (mealType === "lunch") {
+    pluralMeal = mealType + 'es'
+  }
+  else {
+    pluralMeal = mealType + 's'
+  }
+  $(`.${pluralMeal}`).prepend(`
+    <article class='${mealType}' data-${mealType}-food-id=${food.id}>` + addToNameCal(food) +
+    `<p class='delete'><button data-${mealType}-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
     </article>`
   )
 }
 
-const getBreakfastCalorieSum = (meal) => {
+const getCalorieSum = (meal) => {
   var total = 0
-  var sum = meal.foods.forEach(function(food) {
+  var sum = meal.foods.forEach(function (food) {
     total += food.calories
   })
-  appendToBreakfastCalories(total)
-}
-
-const appendToBreakfastCalories = (total) => {
-  $('.breakfast-calorie-sum').append(total);
+  appendToCalories(total, meal)
 }
 
-const getLunchCalorieSum = (meal) => {
-  var total = 0
-  var sum = meal.foods.forEach(function(food) {
-    total += food.calories
-  })
-  appendToLunchCalories(total)
+const appendToCalories = (total, meal) => {
+  $(`.${meal.name.toLowerCase()}-calorie-sum`).append(total);
 }
 
-const appendToLunchCalories = (total) => {
-  $('.lunch-calorie-sum').append(total);
-}
-
-const getDinnerCalorieSum = (meal) => {
+const getCalorieRemaining = (meal) => {
+  if (meal.name === "Breakfast") {
```

Comment:
I think it might be good to pull this out to a method- called Goal and then reference it in this method. Seems like a lot going on here

Reply:
ok great- i implemented!